pkg/util/kube.go

```diff
@@ -419,6 +419,38 @@ func GetPodName(n, labelSelector string) (pod string, err error) {
 	return
 }
 
+// Check PodReady (Example: 2/2 container running on a pod) returns true if the pod is ready. Params: namespace, pod label and timeout to check ready pod.
+func CheckPodReady(n, labelSelector string, timeout int) (ready bool, err error) {
```

Comment:
```suggestion
func CheckPodReady(ns, selector string, timeout time.Duration) (bool, error) {
```
Then you don't need the comment above ;)